crime-commons-mod-schemas/src/main/resources/schemas/evidence/apiUpdateIncomeEvidenceResponse.json

```diff
@@ -26,6 +16,8 @@
       "description": "The date that all of the evidence was received"
     }
   },
-  "additionalProperties": false,
-  "required": ["applicantEvidenceItems", "partnerEvidenceItems", "dueDate"]
+  "extends": {
+    "$ref": "apiCreateIncomeEvidenceResponse.json"
```

Comment:
Should we still be enforcing in here that the `applicantDetails` and `partnerDetails` (now defined in the create response) are required properties?

Reply:
there is no field called `partnerDetails` on the schema. The `applicantDetails` field is marked as required on the `apiIncomeEvidenceItems.json`.